samples/unit-tests/datalabels/i18n/demo.js

```diff
@@ -0,0 +1,55 @@
+QUnit.test('RTL characters on data labels', function (assert) {
```

Comment:
The text outline is general to all labels, not only data labels. So instead of creating a full chart instance in order to run a test on its labels, we should run the test on the `SVGRenderer.text` API directly. Rewrite the test and move it to `unit-tests/svgrenderer/text/demo.js`.